packages/website/ts/index.tsx

```diff
@@ -46,6 +48,12 @@ const LazyConnectDocumentation = createLazyComponent('Documentation', async () =
 );
 
 ReactGA.initialize('UA-98720122-1');
+window.onload = () => {
+    const providerName = (window as any).web3
+        ? utils.web3ProviderToString((window as any).web3.currentProvider)
```

Comment:
I think you want to wrap this and check if web3 is defined or not. `!_.isUndefined((window as any).web3` before trying to access `currentProvider` and ``currentProvider.constructor.name`